src/solve.jl

```diff
@@ -369,4 +369,10 @@ function __init__()
 		end
 	end
 end
-  
\ No newline at end of file
+
+function __solve(prob::OptimizationProblem, opt::Union{Function, Type{<:MOI.AbstractOptimizer}, MOI.OptimizerWithAttributes})
+    optimizer = MOI.instantiate(opt)
+    MOI.add_variables(optimizer, length(prob.x)) 
```

Comment:
`x = ...`

Reply:
Wouldn't that be something like `eval_objective`?

Reply:
What do you mean, I'm just saying that you should store the variables in a variable so that you can retrieve the solution afterward

Reply:
Oh sorry, I thought you looked at other methods for `__solve` and were suggesting `x` same as used in them. Got it now.